src/StringUtils.h

```diff
@@ -223,7 +223,7 @@ bool StringToInteger(const AString & a_str, T & a_Num)
 /** Returns an integer from a key-value string map.
```

Comment:
I think it still is. It means `an integer` as any integral type, not as `int` specifically.
Would changing it to `a number` or `an integral number` be better?